apps/trackers/tests/test_common.py

```diff
@@ -448,6 +448,48 @@ def test_bugzilla_virtualizaiton(self, ps_component):
             in tqb.description
         )
 
+    @pytest.mark.parametrize("is_contract_priority", [True, False])
+    def test_contract_priority(self, is_contract_priority):
+        """
+        test contract priority description generation
+        """
+        ps_module = PsModuleFactory(bts_name="bugzilla")
+        ps_update_stream = PsUpdateStreamFactory(ps_module=ps_module)
+        if is_contract_priority:
+            ContractPriority(ps_update_stream=ps_update_stream.name).save()
+        flaw = FlawFactory()
+        affect = AffectFactory(
+            flaw=flaw,
+            affectedness=Affect.AffectAffectedness.AFFECTED,
+            resolution=Affect.AffectResolution.DELEGATED,
+            ps_module=ps_module.name,
+            ps_component="kvm",
+        )
+        tracker = TrackerFactory(
+            affects=[affect],
+            embargoed=affect.flaw.embargoed,
+            ps_update_stream=ps_update_stream.name,
+            type=Tracker.TrackerType.BUGZILLA,
+        )
+
+        tqb = TrackerQueryBuilder()
+        tqb.instance = tracker
+
+        description = """\
+Contract priorities are scoped by at the \
+product/version level and include all severities \
+and all components within that product/version. \
+All severities (including low) must be resolved \
+within the allotted SLA time."""
```

Comment:
You can use sth like this to keep the indentation (this will not include any extra blanks or newlines):

```
description = (
    "word1"
    "word2"
)
```

Reply:
I've fixed, I've also changed other tests in this file to follow the same pattern in a separated commit, please take  a look again.